lib/modules/authentication_module/view/login_page.dart

```diff
@@ -42,7 +41,9 @@ class LoginPage extends GetView<LoginController> {
             const CustomDivider(),
```

Comment:
Name this CZNTechDivider. For all custom components, instead of using "Custom" use "CZNTech"

Reply:
Always name your components in camelCase